express/src/controllers/manage-account.ts

```diff
@@ -201,4 +210,72 @@ export const changePassword = async function (req: Request, res: Response) {
     }
     req.session.updatedField = "password";
     res.redirect('/account');
+}
+
+export const showChangePhoneNumberForm = async function (req: Request, res: Response) {
+    res.render('account/change-phone-number.njk', {values: {mobileNumber: req.session.mobileNumber}});
```

Comment:
```suggestion
    res.render('account/change-phone-number.njk');
```
Remove to not pre-fill the form

Reply:
👍🏻

Reply:
🔥